src/gui/sdlmain.cpp

```diff
@@ -4792,6 +4792,52 @@ static void maybe_write_primary_config(const CommandLineArguments* args)
 	}
 }
 
+static std::optional<int> maybe_handle_command_line_output_only_actions(
+        const CommandLineArguments* args, char* argv[])
```

Comment:
Again, I prefer a reference instead if pointer. And we don't need the whole `char* argv[]` inside - passing `argv[0]` (program name) should be enough.

Reply:
Good points, done both.